src/index.js

```diff
@@ -44,6 +54,7 @@ export default (target, safeOverJump = false) => {
     const lastKey = keys.pop();
     let tmp = target;
     keys.forEach(key => {
+      if (isPrototypePolluted(key)) return;
```

Comment:
Do you think is better returning a `false` object instead of void?

Reply:
`false` would've been a better option.